packages/schema-registry-api/src/api.ts

```diff
@@ -33,7 +33,14 @@ const apiFetch = async <T>(req: string | Request, init: RequestInit = {}): Promi
     },
   };
   const res = await fetch(req, { ...defaultInit, ...init });
-  const data = await res.json();
+  let data;
+  try {
+    data = await res.json();
+  } catch (e) {
```

Comment:
A bit pedantic but keeps things consistent with the rest of the file

```suggestion
  } catch (error) {
```